javascript/javascript1/week4/Voice-assistant.js

```diff
@@ -0,0 +1,117 @@
+let name = ""
+const todoList = []
+
+function getReply(command) {
+ if (typeof command !== "string" || command === "") {
+   return "Please enter a valid command"
+ }
+/* let num1 = x;
+let num2 = y;*/
+ switch(command){
+
+   case "Hello my name is Benjamin":
+     name = "Benjamin"
+       return `Nice to meet you Benjamin`
+
+   case "What is my name?":
+       if(name !== ""){
+         return "You didn't say your name yet"
+       }
+   case "Add Fishing to my todos":
+        if (command !== "") {
```

Comment:
You should check if the command includes for example `to my todos` and then find which todo you should push to the `todoList`
Hint: https://www.w3schools.com/jsref/jsref_includes.asp and https://www.w3schools.com/jsref/jsref_substring.asp

Reply:
You can also write a function for it and then use it for every case where you have `Add ____ to my todos`